Zend/tests/constexpr/new_invalid_operation_in_arg.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-Invalid operation in new arg in const expr
+Valid operation in new arg in const expr
```

Comment:
This one is interesting, because AFAICT it no longer is a constexpr context since https://wiki.php.net/rfc/arbitrary_static_variable_initializers

Reply:
Indeed, guess I forgot to update the test description.